example/lib/payment_slection_page.dart

```diff
@@ -217,7 +217,7 @@ class _PaymentSelectionPageState extends State<PaymentSelectionPage> {
           merchant can use this response["data"] upiAccounts or can again call
           _razorpay.getLinkedUpiAccounts(turboUPIModel?.mobileNumber)
        */
-        Navigator.pop(context);
+        //Navigator.pop(context);
```

Comment:
Commented out to prevent crashing the app:  Since popping the activity would nullify the context, we're preserving it for future use.